res/css/views/auth/_AuthBody.scss

```diff
@@ -94,6 +94,22 @@ limitations under the License.
             background-color: $authpage-focus-bg-color;
         }
     }
+
+    .mx_recaptchaContainer {
+        .mx_recaptchaContainer_info,
+        .mx_recaptchaContainer_error {
+            margin: 12px auto;
+        }
+    }
+
+    .mx_AuthBody_link_goBack {
+        margin: 12px auto 6px;
```

Comment:
Please use the spacing variables for standard spacings https://github.com/matrix-org/matrix-react-sdk/blob/develop/res/css/_spacing.scss

Reply:
Would you let me know what kind of spacing requires those variables? I'm asking this for future reference and because `@spacing-*` is currently used only in 12 files.

Before changing those values, I would like to ask you to tell me whether it is not obsolete.

If they are the recommended way of spacing, I would like to work on that with another PR to start standardization. The first step should be fairly easy as you just need to do find and replace.

Reply:
Please let me now if using those variables is officially recommended. I would open an issue for tracking the task to replace nonstandard ones with standard ones. Thanks.